components/03-modules/cart-list-item/CartListItem.stories.js

```diff
@@ -0,0 +1,65 @@
+import { storiesOf } from '@storybook/vue'
+import { action } from '@storybook/addon-actions'
+
+import App from '../../01-globals/app/App.vue'
+import AlpacaCartListItem from './CartListItem.vue'
+
+import cartListItem from './mocks/cartListItem.json'
+
+storiesOf('Modules/Cart list item', module)
+  .add('Default', () => ({
```

Comment:
Do we need 2 stories `Default` and `With links`? What is different?

Reply:
In case `With links` remove and edit buttons are `a` tags.